static/js/settings_bots.js

```diff
@@ -523,6 +523,10 @@ export function set_up() {
                                         data.avatar_url + "&v=" + image_version.toString(),
                                     );
                             }
+                            if (bot.owner_id !== bot_owner_id) {
+                                bot_data.del(bot_id);
+                                render_bots();
+                            }
```

Comment:
This belongs in the `server_events_dispatch` code path -- otherwise it won't live update other browser windows that might be viewing bot settings at the time (for the current user or other users).

Doing it correctly may require a bit of care to only rerender this panel if it's already displayed, but it's worth the extra effort to be correct.

(This is covered in our new feature tutorial if you want more background).